test/blob_test.js

```diff
@@ -1,47 +1,57 @@
-suite.only("Azure Blob", function() {
+suite("Azure Blob", function() {
   var azure   = require('../');
   var assert  = require('assert');
+  var utils   = require('../lib/utils');
 
   // Create azure blob client
   var blob = new azure.Blob({
     accountId: process.env.AZURE_STORAGE_ACCOUNT,
     accessKey: process.env.AZURE_STORAGE_ACCESS_KEY
   });
 
+  var anonymousBlob = new azure.Blob({
+    accountId: process.env.AZURE_STORAGE_ACCOUNT,
```

Comment:
ahh, right, this is why you need to specify the accountId even for anonymous access!!!  Ignore my comment above about that